packages/expo-router/src/link/Link.tsx

```diff
@@ -33,6 +33,14 @@ export function Redirect({ href }: { href: Href }) {
 /**
  * Component to render link to another route using a path.
  * Uses an anchor tag on the web.
+ *
+ * @param props.href Absolute path to route (e.g. `/feeds/hot`).
```

Comment:
Adding `Link` props in multiple places might conflict with the recent and upcoming docs changes.

The props are already picked and displayed in in the current unversioned Router API reference: https://docs.expo.dev/versions/unversioned/sdk/router/

![CleanShot 2024-10-29 at 16 46 24](https://github.com/user-attachments/assets/cbe802c4-bc0f-4f35-b615-59b402a26bb2)

I've also a [PR](https://github.com/expo/expo/pull/32329) open that adds more context/examples from current way of how props are exposed: http://docs.expo.dev-pr-32329.s3-website-us-east-1.amazonaws.com/versions/unversioned/sdk/router/#link